clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1985,7 +1985,8 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
 
 def ReqdWorkGroupSizeAttrDocs : Documentation {
   let Category = DocCatFunction;
-  let Heading = "cl::reqd_work_group_size, intel::reqd_work_group_size";
+  let Heading = "GNU<reqd_work_group_size>, CXX11<cl::reqd_work_group_size, 
+                CXX11<intel::reqd_work_group_size>";
```

Comment:
```suggestion
  let Heading = "__attribute__((reqd_work_group_size)), [[cl::reqd_work_group_size]], 
                [[intel::reqd_work_group_size]]";
```

Reply:
buildbot/Build_PR_Win didn't pass because Heading, and writes:
"D:/buildbot/Product_worker_intel/Build_PR_With_Lit_Win/llvm.src/clang/include\clang/Basic/AttrDocs.td:2008:17: error: Unknown token when parsing a value
  let Heading = "__attribute__((reqd_work_group_size)), [[cl::reqd_work_group_size]],"

Reply:
Maybe just "reqd_work_group_size" will be better?

Reply:
Just "reqd_work_group_size" works for me

> unknown token when parsing a value
let Heading = "attribute((reqd_work_group_size)), [[cl::reqd_work_group_size]],"

Probably this is some kind of keyword which is being parsed regardless of the fact that it is within a string literal